src/Parser/Core/Modules/Others/DistanceMoved.js

```diff
@@ -66,13 +73,13 @@ class DistanceMoved extends Analyzer {
   }
 
   statistic() {
-    debug && console.log(`Total distance moved: ${this.totalDistanceMoved} yds`);
+    debug && console.log(`Time spend moving: ${this.timeSpendMoving / 1000} s, Total distance moved: ${this.totalDistanceMoved} yds`);
     return (
       <SmallStatisticBox
         icon={<Icon icon="spell_fire_burningspeed" />}
-        value={`≈${formatThousands(this.totalDistanceMoved)} yards`}
-        label="Distance moved"
-        tooltip={`≈${formatThousands(this.totalDistanceMoved / (this.owner.fightDuration / 1000) * 60)} yards per minute. Consider this when analyzing the fight, as some fights require more movement than others. Unnecessary movement can result in a DPS/HPS loss.`}
+        value={`≈${formatPercentage(this.timeSpendMoving / (this.owner.fightDuration))}%`}
+        label="Fight spend moving"
+        tooltip={`In ≈${formatThousands(this.timeSpendMoving / 1000)} seconds of movement you moved by ≈${formatThousands(this.totalDistanceMoved)} yards. Consider this when analyzing the fight, as some fights require more movement than others. Unnecessary movement can result in a DPS/HPS loss.`}
       />
```

Comment:
``you moved X yards. `` would sound better than ``you moved by X yards``

Reply:
Indeed it would, addressed.